### PR TITLE
completed frontend structure: created directories with corresponding …

### PR DESCRIPTION
**Pull request by gencalada**
…files, created app.js routes, added mockPigion array to app.js, and basic styling

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <div className='footer'>
+      
+      <h1>Footer</h1>
```
Comment (elysemontano) [inline src/components/Footer.js:7]: I would suggest updating this with content that you actually want in the application like copyright and your names.

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          >
+            Link
+          </NavLink>
+        </NavItem>
+        <NavItem>
+          <NavLink href="#">
+            Another Link
+          </NavLink>
+        </NavItem>
+      </Nav>
```
Comment (elysemontano) [inline src/components/Header.js]: I would also suggest updating this so that your navigation is usable and not just filler text.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left some comments on your Header and Footer.

File: src/components/Footer.js (hunk 1 above)
Comment (gencalada) [reply src/components/Footer.js:7]: Okay, got it! getting right to it.

File: src/components/Header.js (hunk 2 above)
Comment (gencalada) [reply src/components/Header.js]: okay, thank you! We just overlooked the filler text 😅

**Review by elysemontano — APPROVED** [review]

**Pull request merged**